include/ck/ck.hpp

```diff
@@ -4,12 +4,19 @@
 #pragma once
 
 #include "ck/config.h"
+#include "ck/utility/env.hpp"
```

Comment:
_from https://github.com/ROCm/MIOpen/pull/2961#issuecomment-2117552920_

This exports its a macro with generic name (`ENV`) in CK's public API. Please take care about possible name collisions, for example by adding and unique prefix (like `CK_` or `COMPOSABLE_KERNELS_`). Or, better, include env.hpp only where it is actually required.

Reply:
Oops. I'll replace ENV with CK_ENV in a separate PR.
At present we check the logging environment variable in ~26 files, so I can move the env.hpp there. But in the long term, if more logging is added, we may need it all over the place.

Reply:
Submitted new PR: https://github.com/ROCm/composable_kernel/pull/1296